### ISSUE TITLE
Translation Platform

### ISSUE THREAD

**Issue by dskzero22** [sosauce/CuteMusic#5]
I would like to start translating CuteMusic into Spanish, adding a translation platform like Crowdin would be very helpful to be able to do it :)

Greetings and good luck, Although it's the initial launch of your app and there are probably still things to improve or add, I liked it a lot, I hope everything goes well here hehe

**Comment by sosauce** [comment]
Good Idea ! I should do french ! It's not my top priority for now, Will ping you when I add translations for sure !!

**Comment by sosauce** [comment]
Also do you have discord ? because I have ZERO clue hoe to use crowdin ahah ..

**Comment by dskzero22** [comment]
> Also do you have discord ? because I have ZERO clue how to use crowdin ahah ..

If I have to be honest, I use it very little, I use Telegram more as my main network, and well, emails. But I'll try to join your support server later n.n

**Comment by sosauce** [comment]
Okay thanks :)

**Comment by sosauce** [comment]
PLEASE DO NOT COMMENT ANYTHING UNDER THIS (Will get removed)

Translation platform is NOT opened for now (any help on how todo so is welcome.)

The below languages will get their translations when avaible :

- French (me)
- Spanish @dskzero22 
- Bangla @weird-budy

**Comment by sosauce** [comment]
Update: If you are willing to help using a translation platform (their main instance, don't come in trying to promote a niche unknown instance because I need something reliable) feel free to open an issue and or text me on [Discord](https://discord.gg/c6aCu4yjbu)

**Comment by sosauce** [comment]
https://hosted.weblate.org/engage/chocola/
